db/migrate/20210414150758_change_non_uk_degree_non_naric_to_non_enic.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class ChangeNonUkDegreeNonNaricToNonEnic < ActiveRecord::Migration[6.1]
+  NON_ENIC = "non_enic"
```

Comment:
Wondering if we just use the constant already defined in the initialiser?

Reply:
@darokel - I initially didn't want the migration to depend on any of the rest of the code. If we changed the name of the constant in the initialiser again in the future, then this migration would break for someone running all migrations afresh.